docs/admin-guides/managing-optimizers.md

```diff
@@ -84,11 +114,19 @@ The following configuration needs to be filled in:
+|---------------------|----------------|----------|-----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| scheduling-policy   | All | No       | quota                 | The scheduler group scheduling policy, the default value is `quota`, it will be scheduled according to the quota resources configured for each table, the larger the table quota is, the more optimizer resources it can take. There is also a configuration `balanced` that will balance the scheduling of each table, the longer the table has not been optimized, the higher the scheduling priority will be. |
+| memory   | local | Yes      | N/A                   | The memory size of the local optimizer Java process.                                                                                                                                                                                                                                                                                                                                                             |
+| jobmanager.memory | flink | No       | N/A                   | The memory size of flink jobmanger, it will overwrite the `jobmanager.memory.process.size`.                                                                                                                                                                                                                                                                                                                      |
+| taskmanager.memory | flink | No | N/A                   | The memory size of flink taskmanger, it will overwrite the `taskmanger.memory.process.size`.                                                                                                                                                                                                                                                                                                                     |
+
+Tips, the following properties of OptimizerContainer could be overwritten by group properties.
+
+| Properties         | Container Type | Description                                   |
```

Comment:
Is it necessary to split it into different tables for display here?

Reply:
Here is properties guide for OptimizerGroup, the above table is properties guide for OptimizerContainer.